go-controller/pkg/util/net_linux.go

```diff
@@ -302,24 +302,23 @@ func LinkRoutesAdd(link netlink.Link, gwIP net.IP, subnets []*net.IPNet, mtu int
 	return nil
 }
 
+// LinkRoutesApply applies routes for given subnets.
+// For each subnet it searches for an existing route by destination(subnet) on link:
+// * if found and gwIP, mtu or src changed the route will be updated
+// * if found and gwIP, mtu and src were not changed it does nothing
```

Comment:
nit: I think this second bullet for me is confusing and not needed, the first bullet already says that "if found AND one of the fields changes" only then route gets updated, so either I would move the "else it does nothing to the first point itself  OR just not say anything about the "does nothing" part since its understood from point1 - that will go well with the if else conditions we have in the code.

Reply:
I removed the second line, I agree that the over explaining is confusing.